reco_utils/nni/svd_training.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think we should consider having a folder where we put all the code related to tuning, instead of `reco_utils/nni`, if tomorrow we have another hypterparameter tunning framework, I think the repo can get messy. Alternatively, we could have the nni code inside evaluation. This is were @yueguoguo added the other hyperparameter evaluation

Reply:
I don't think evaluation is the right place; after all tuning is a different thing. It would be more appropriate to have a folder called tuning and, under that folder, azureml and nni and the spark tuning code. This would require testing again all the tuning notebooks, so I would leave it for a separate PR. Hyperdrive and NNI rely on several path names, so they will break as soon as you change the folder structure.

Reply:
that makes sense, added issue here: https://github.com/Microsoft/Recommenders/issues/689

Reply:
Having a dedicated folder for tuning makes perfect sense.